README.md

```diff
@@ -80,10 +87,18 @@ if isinstance(result, Success):
 _Note: when sending an empty value (`""`) as value for a property, the property will be deleted._
 
 ```python
+from journyio.account_identified import AccountIdentified
+
+account = AccountIdentified("accountId", "www.domain.tld")
+# or
+account = AccountIdentified.by_account_id("accountId")
+# or
+account = AccountIdentified.by_domain("www.domain.tld")
+
 properties = Properties()
 properties["property1"] = "value1"
 properties["property2"] = ""  # property2 will be deleted
-result = client.upsert_account("accountId", "accountName", properties, ["memberId1", "memberId2"])
+result = client.upsert_account(account, properties, ["memberId1", "memberId2"])
```

Comment:
Let's set `name` in the example properties? Maybe we can add more realistic properties?

```
  properties: {
    total_amount_of_users: 53,
    name: "ACME, Inc",
    is_paying_account: true,
    registered_at: new Date(/* ... */),
    this_property_will_be_deleted: "",
  },
```

Reply:
Great idea, will do!